src/index.js

```diff
@@ -0,0 +1,117 @@
+const inquirer = require("inquirer");
+require("dotenv").config();
+
+const cTable = require("console.table");
+//  import questions
+const {
+  questions,
+  departmentQuestions,
+  employeeQuestions,
+  deleteQuestions,
+  askRoleQuestions,
+  getEmployeeDetails,
+  updateEmployees,
+} = require("./questions");
+
+// get the client
+const mysql = require("mysql2");
+const initDatabase = require("./db");
+
+const init = async () => {
+  try {
+    const { executeQuery, closeConnection } = await initDatabase({
+      host: process.env.DB_HOST,
+      user: process.env.DB_USER,
+      password: process.env.DB_PASSWORD,
+      database: process.env.DB_NAME,
+    });
+    let inProgress = true;
+
+    while (inProgress) {
+      // get answers for first set of questions
+      const answers = await inquirer.prompt(questions);
+
+      if (answers.proceed === "View all departments") {
+        const departments = await executeQuery("SELECT * FROM department");
```

Comment:
These queries could be separated into their own files to ensure the index file is kept clean and readable. For example, you could have a file for all of the different queries e.g. `view.js`, `add.js`, `update.js` and so on.